persistence-modules/spring-data-jpa-2/src/main/java/com/baeldung/entitygraph/repository/CharacteristicsRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.entitygraph.repository;
+
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.baeldung.entitygraph.model.Characteristic;
+
+public interface CharacteristicsRepository extends JpaRepository<Characteristic , Long> {

```

Comment:
remove space - don't forget to update the article as well

Reply:
Done